diff --git a/node_modules/vue-place-autocomplete/src/PlaceAutocompleteField.vue b/node_modules/vue-place-autocomplete/src/PlaceAutocompleteField.vue
index 1344c79..552df6a 100644
--- a/node_modules/vue-place-autocomplete/src/PlaceAutocompleteField.vue
+++ b/node_modules/vue-place-autocomplete/src/PlaceAutocompleteField.vue
@@ -1,6 +1,6 @@
 <template>
     <div class="autocomplete-field" @keydown="onKeydown" @keyup="onKeyup">
-        <input-field
+        <input
             v-model="query"
             v-bind-events
             v-bind="$attrs"
@@ -11,9 +11,9 @@
             autocomplete="no"
             @blur="onBlur"
             @focus="onFocus"
-            @input="$emit('input', query)">
+            @input="$emit('input', query)"/>
             <activity-indicator v-if="showActivityIndicator" size="xs" type="spinner"/>
-        </input-field>
+       
         <place-autocomplete-list v-if="predictions && showPredictions" :items="predictions" @item:click="onItemClick" @item:blur="onItemBlur"/>
     </div>
 </template>
